tests/test_app.py

```diff
@@ -25,21 +25,55 @@ def test_timeline(self):
         # TODO test GET and POST Endpoints
         # TODO Test timeline page
```

Comment:
Incomplete TODOs~

Reply:
There are other functions to test the GET and POST end points and the timeline page
I removed the todo comments after merging